compiler/noirc_evaluator/src/ssa/opt/mem2reg.rs

```diff
@@ -902,8 +903,8 @@ mod tests {
 
         let b1_instructions = main.dfg[b1].instructions();
 
-        // We expect the last eq to be optimized out, only the store from above remains
-        assert_eq!(b1_instructions.len(), 1);
+        // We expect the last eq to be optimized out
+        assert_eq!(b1_instructions.len(), 0);
```

Comment:
Seeing as we're touching this test can you convert it to use the parser?

Reply:
switched